muser/README.md

```diff
@@ -0,0 +1,37 @@
+README.md
+muser-firebase-export Build Status
```

Comment:
Please add a .gitignore file to /src/main/resources to prevent files (especially the admin key) from being committed. You can copy/paste:
https://github.com/CUTR-at-USF/travel-behavior-analysis/blob/master/src/main/resources/.gitignore

Reply:
I think we're still missing this?